### PR TITLE
Work with contracts deployed locally- dapp

### PR DESCRIPTION
**Pull request by r-czajkowski**
Closes https://github.com/keep-network/keep-core/issues/1810
Depends on: https://github.com/keep-network/keep-ecdsa/pull/462

This PR adds a guide in the readme file on how to work with contracts that have been deployed to your local network. Additionally removes unused scripts that copied contracts artifacts to the `src/contracts` dir.

### CODE REVIEW

**Comment by pdyraga** [comment]
Is this one still up to date?

**Comment by r-czajkowski** [comment]
The script does a great job 🚀 ! @dimpar, let's also add info about this script and requirements in the readme file.

File: scripts/start_dashboard.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+LOG_END='\n\e[0m' # new line + reset color
+
+cd $KEEP_CORE_SOL_PATH
+
+## uncomment when version of a dependency in package.json has changed.
+# printf "${LOG_START}Installing NPM dependencies...${LOG_END}"
+# rm -f package-lock.json
+# rm -rf node_modules/
+# npm install
+
```
Comment (pdyraga) [inline scripts/start_dashboard.sh:26]: Isn't just `npm install` enough here? We are doing it in all other initialization scripts so I'd leave it uncommented.

Comment (dimpar) [reply scripts/start_dashboard.sh]: yea, I guess we can leave it uncommented. Changed.

**Comment by dimpar** [comment]
@pdyraga @r-czajkowski do you guys have additional comments or ready for merging?

**Comment by dimpar** [comment]
I've addressed your comments @r-czajkowski @pdyraga Ready for another look.

File: solidity/dashboard/README.md [hunk 2]
```diff
@@ -55,6 +55,16 @@ truffle exec ./scripts/delegate-tokens.js
 
 * Use metamask with `localhost:8545` to use Ganache test network. Import your first Ganache test account into metamask and you should be able to see the demo data.
 
+### Work with contracts deployed locally
+
```
Comment (r-czajkowski) [inline solidity/dashboard/README.md:59]: Maybe it's worth mentioning that we need to clone a keep-ecdsa and tbtc repo 🤔.

Comment (dimpar) [reply solidity/dashboard/README.md:59]: sure, done

**Comment by r-czajkowski** [comment]
looks good +1. Let's merge it!

**Pull request merged**